eland/series.py

```diff
@@ -442,6 +444,11 @@ def __gt__(self, other: Union[int, float, "Series"]) -> BooleanFilter:
             return ScriptFilter(painless, lang="painless")
         elif isinstance(other, (int, float)):
```

Comment:
So what we can do here is right at the top of `__gt__()` and other methods:

```python
if isinstance(other, np.datetime64):
    other = pd.to_datetime(other)
...
```

The default elasticsearch-py serializer handles datetime microseconds. If you think we should handle timezones that can be done in a different PR.

Reply:
This is really convenient, I will change this!